AppCenter/AppCenter/Internals/Model/MSLog.h

```diff
@@ -30,6 +30,11 @@
  */
 @property(nonatomic) MSDevice *device;
 
+/**
+ * Transient object tag.
```

Comment:
I am a little confused about the tag thing, what is it used for? Is it really necessary?

Reply:
OK we are using it to remember which target is this log tied to. Should probably be more explicit about it.

Reply:
Also there is another issue, in the future an event could be sent to multiple targets, for instance the session log could be sent to all existing targets. This is why targets are stored into a set.

Reply:
Tag is a general data field that can be set to anything. For instance, it can be set to a list of transmission targets. Then you could check if it exists. Alternatively, we could make the tag into a list of objects. That probably depends on future discussions.

As far as being more explicit, I think I opted to align with the Android implementation. In this case, I don't think we should be inconsistent. Also, we could use it for other reasons in the future (maybe some kind of PII scenario?) so I like that it is general.

Reply:
It can be general purpose but it would require an example to me in the comment.

Reply:
I think this is a good idea.

Reply:
I'll add the example we currently use to the comment